### DISCUSSION TITLE
[欧皇] 看看我的驱动盘

### DISCUSSION THREAD

**Discussion by Lynette-CN** [share121/inter-knot#901] [其他]
![Uploading abb1046c2cd5eb7167ca33f5c25d622.jpg…]()

**Comment by Lynette-CN** [comment]
![Uploading abb1046c2cd5eb7167ca33f5c25d622.jpg…]()

**Comment by Dave-12138** [comment]
uploading意思是图片正在上传，你要等uploading消失后再点comment

**Comment by themagicmoon** [comment]
兄弟图没发出来

**Comment by Xmug** [comment]
![俺的图图呢](https://github.com/user-attachments/assets/0caf6acf-50a7-4e5a-9265-c9cd1da89b23)
